proto/proto-backplane-grpc/src/main/proto/deephaven/proto/table.proto

```diff
@@ -500,15 +505,29 @@ message MergeTablesRequest {
 
 message SnapshotTableRequest {
```

Comment:
Please regenerate gwt bindings so we can fix the JS API calls for table.freeze() and table.snapshot().

Reply:
Will do.

Reply:
Done. Can you look at the relevant JS commits?